### PR TITLE
SG-1149: Fixes a js error when cookie is not used in form.io forms

### PR DESCRIPTION
**Pull request by henryjiang-sfgov**
Problem as stated above, steps to reproduce: 

- Visit [https://test-sfgov.pantheonsite.io/apply-for-mini-grant](https://test-sfgov.pantheonsite.io/apply-for-mini-grant)
- Inspect the console to find the js error: `Uncaught (in promise) TypeError: Cannot read property 'cookies' of undefined`

Fixed in `paragraph--form-io.html.twig`
- Visit [http://pr-579-sfgov.pantheonsite.io/apply-for-mini-grant](http://pr-579-sfgov.pantheonsite.io/apply-for-mini-grant)
- Inspect the console to find no js error.

### CODE REVIEW

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--form-io.html.twig [hunk 1]
```diff
@@ -88,7 +88,6 @@
               if (options.sfoptions && options.sfoptions.hide instanceof Object) {
                 customHideElements(options.sfoptions.hide)
               }
-              console.log(submission.data)
```
Comment (jacine) [inline web/themes/custom/sfgovpl/templates/paragraphs/paragraph--form-io.html.twig:91]: @henryjiang-sfgov Hey, there is another console.log on line 173.  Did you mean to keep this one in? 

```js
console.log("setting cookie for " + item + " with value " + this.value)
```

Noticed this error too, when updating my branch, so testing the PR now. 😄

**Comment by henryjiang-sfgov** [comment]
> Can confirm this resolves the error. Thanks!

Thanks for the quick reply!

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--form-io.html.twig (hunk 1 above)
Comment (henryjiang-sfgov) [reply web/themes/custom/sfgovpl/templates/paragraphs/paragraph--form-io.html.twig:91]: aha yes, sorry I meant to keep this around for a bit until things settle down.

Comment (jacine) [reply web/themes/custom/sfgovpl/templates/paragraphs/paragraph--form-io.html.twig:91]: Got it!  Thanks for letting me know! 😄

**Pull request merged**